/**: # wildcard for the node name
  ros__parameters:
    channels:
      torque_x: 0
      torque_y: 1
      torque_z: 2
      thrust_x: 3
      thrust_y: 4
      thrust_z: 5

    # OUTPUT = MixerMatrix * INPUT
    # 8x1 = 8x8 * 8x1

    mixer_matrix: [
      +0.25, -0.25, +0.25, -0.25, +0.00, +0.00, # motor 0
      +0.25, +0.25, +0.25, +0.25, +0.00, +0.00, # motor 1
      +0.25, +0.25, -0.25, -0.25, +0.00, +0.00, # motor 2
      +0.25, -0.25, -0.25, +0.25, +0.00, +0.00, # motor 3
      +0.00, +0.00, +0.00, +0.00, +0.00, +0.00, # motor 4
      +0.00, +0.00, +0.00, +0.00, +0.00, +0.00, # motor 5
      +0.00, +0.00, +0.00, +0.00, +0.00, +0.00, # motor 6
      +0.00, +0.00, +0.00, +0.00, +0.00, +0.00, # motor 7
    ]

    limit_matrix: [
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 0
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 1
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 2
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 3
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 4
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 5
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 6
     +1.0, +1.0, +1.0, +1.0, +0.0, +0.0, # motor 7
    ]
    # Thrust is assumed to be a quadratic function of revs per second.
    quadratic_coefficient: 2.6e-3
    linear_coefficient: 4.2e-2
    max_rotations_per_second: 50.0
